src/player.cpp

```diff
@@ -3073,15 +3058,32 @@ Strength - 4;    Dexterity - 4;    Intelligence - 4;    Perception - 4"));
             fold_and_print( w_info, 0, 1, FULL_SCREEN_WIDTH - 2, c_magenta, s );
             wrefresh(w_info);
 
+            // which of the four paired body parts should get combined?
+            // TODO deduplicate this with print_encumbrance()
+            bool combined_limbs[4] = {}; // all false
```

Comment:
Sure. This would be much easier to read and understand. And it's probably much safer for future changes (for example when eyes get split into two body parts).

Also: don't use a literal number as array size, use `num_bp` or whatever it is supposed to represent.